3.33.0

trentm opened the pull request:
Cinco de Mayo!

This is a quick release to:
- get a fix out for #2670 
- test out the new Jenkinsfile-based `npm install ...` handling

apmmachine commented:
## :green_heart: Build Succeeded


<!-- BUILD BADGES-->
> _the below badges are clickable and redirect to their specific view in the CI or DOCS_
[![Pipeline View](https://img.shields.io/badge/pipeline-pipeline%20-green)](https://apm-ci.elastic.co/blue/organizations/jenkins/apm-agent-nodejs%2Fapm-agent-nodejs-mbp%2FPR-2676/detail/PR-2676/1//pipeline) [![Test View](https://img.shields.io/badge/test-test-green)](https://apm-ci.elastic.co/blue/organizations/jenkins/apm-agent-nodejs%2Fapm-agent-nodejs-mbp%2FPR-2676/detail/PR-2676/1//tests) [![Changes](https://img.shields.io/badge/changes-changes-green)](https://apm-ci.elastic.co/blue/organizations/jenkins/apm-agent-nodejs%2Fapm-agent-nodejs-mbp%2FPR-2676/detail/PR-2676/1//changes) [![Artifacts](https://img.shields.io/badge/artifacts-artifacts-yellow)](https://apm-ci.elastic.co/blue/organizations/jenkins/apm-agent-nodejs%2Fapm-agent-nodejs-mbp%2FPR-2676/detail/PR-2676/1//artifacts) [![preview](https://img.shields.io/badge/docs-preview-yellowgreen)](http://apm-agent-nodejs_2676.docs-preview.app.elstc.co/diff)  [![preview](https://img.shields.io/badge/elastic-observability-blue)](https://ci-stats.elastic.co/app/apm/services/apm-ci/transactions/view?rangeFrom=2022-05-05T18:26:32.218Z&rangeTo=2022-05-05T18:46:32.218Z&transactionName=BUILD+apm-agent-nodejs%2Fapm-agent-nodejs-mbp%2FPR-%7Bnumber%7D&transactionType=job&latencyAggregationType=avg&traceId=ae69189d18096e7675a7d5459b31a7bb&transactionId=0217a27c27d2b780)

<!-- BUILD SUMMARY-->
<details><summary>Expand to view the summary</summary>
<p>

#### Build stats




* Start Time: 2022-05-05T18:36:32.218+0000


* Duration: 20 min 13 sec




#### Test stats :test_tube:

| Test         | Results                         |
| ------------ | :-----------------------------: |
| Failed       | 0  |
| Passed       | 249923  |
| Skipped      | 0 |
| Total        | 249923   |


</p>
</details>

<!-- TEST RESULTS IF ANY-->


<!-- STEPS ERRORS IF ANY -->












## :robot: GitHub comments

To re-run your PR in the CI, just comment with:



- `/test` : Re-trigger the build.


- `run module tests for <modules>` : Run TAV tests for one or more modules, where `<modules>` can be either a comma separated list of modules (e.g. memcached,redis) or the string literal ALL to test all modules


- `run benchmark tests` : Run the benchmark test only.


- `run` `elasticsearch-ci/docs` : Re-trigger the docs validation. (use unformatted text in the comment!)




<!--COMMENT_GENERATED_WITH_ID_comment.id-->

trentm commented:
tag pushed.

https://apm-ci.elastic.co/job/apm-agent-nodejs/job/apm-agent-nodejs-mbp/view/tags/job/v3.33.0/ building

trentm commented:
The automatic npm publish failed. Discussion at https://github.com/elastic/apm-agent-nodejs/issues/2625

In the meantime, I'll manually npm publish:

```
% npm publish
...
elastic-apm-node@3.33.0
```

and do the opbeans bump-version manually:

```
[13:18:33 trentm@pink:~/el/opbeans-node (git:main)]
% .ci/bump-version.sh 3.33.0
+ AGENT_VERSION=3.33.0
++ pwd
+ docker run --rm -t --user 501 -e HOME=/tmp -e AGENT_VERSION=3.33.0 -w /app -v /Users/trentm/el/opbeans-node:/app node:12-alpine /bin/sh -c 'set -x
    CI=true npm install --ignore-scripts elastic-apm-node@3.33.0'
+ CI=true npm install --ignore-scripts elastic-apm-node@3.33.0
npm WARN deprecated async-cache@1.1.0: No longer maintained. Use [lru-cache](http://npm.im/lru-cache) version 7.6 or higher, and provide an asynchronous `fetchMethod` option.
+ elastic-apm-node@3.33.0
removed 2 packages, updated 1 package and audited 466 packages in 30.729s

74 packages are looking for funding
  run `npm fund` for details

found 8 vulnerabilities (1 low, 5 moderate, 2 critical)
  run `npm audit fix` to fix them, or `npm audit` for details
+ sed -ibck 's#\(org.label-schema.version=\)\(".*"\)\(.*\)#\1"3.33.0"\3#g' Dockerfile
+ rm -f Dockerfilebck
+ git add package.json package-lock.json Dockerfile
+ git commit -m 'fix(package): bump elastic-apm-node to v3.33.0'
[main 30eac98] fix(package): bump elastic-apm-node to v3.33.0
 3 files changed, 5 insertions(+), 19 deletions(-)
[13:19:28 trentm@pink:~/el/opbeans-node (git:main)]
% git push origin HEAD
...
To github.com:elastic/opbeans-node.git
   15a9609..30eac98  HEAD -> main
[13:19:51 trentm@pink:~/el/opbeans-node (git:main)]
% git tag v3.33.0
[13:20:20 trentm@pink:~/el/opbeans-node (git:main)]
% git push origin !$
git push origin v3.33.0
Total 0 (delta 0), reused 0 (delta 0), pack-reused 0
To github.com:elastic/opbeans-node.git
 * [new tag]         v3.33.0 -> v3.33.0
```
